creating-virtual-machines/interfaces-and-networks.adoc

```diff
@@ -238,15 +238,68 @@ spec:
     pod: {}
 ----
 
+______________________________________________________________________________
+**Note:** If a specific MAC address is configured in the virtual machine
+interface spec, it's passed to underlying CNI plugin that should configure
+backend to allow for this particular address. While some plugins are known to
+work without special support for custom MAC addresses, others do (for example,
+OpenShift SDN needs to configure OVS flows to allow traffic meant for the
+chosen MAC address).
+______________________________________________________________________________
+
+______________________________________________________________________________
+**Note:** If a plugin doesn't support `mac` attribute defined by Kubernetes
```

Comment:
This is confusing. In the previous section you say some are know to work without special effort. @SchSeba could correct me here, but I think that most plugins are OK with custom address (unlike OpenShiftSDN), but the address must be set using the tuning CNI.

Reply:
Plugin not handling `mac` attribute natively == plugin doesn't support it. That you can then work it around with `tuning` plugin is just a function of the original plugin relying on netlink devices only, or not. I will think how to reword it.